tutorials/spectral-cube-reprojection/SpectralCubeReprojectExample.ipynb

```diff
@@ -0,0 +1,651 @@
+{
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #1.    filename_1 = download_file("https://almascience.nrao.edu/dataPortal/member.uid___A001_X1465_X3a33.BrickMaser_sci.spw71.cube.I.manual.image.pbcor.fits",
</pre><p>Maybe this should use <code>astroquery.alma</code> instead? Might that be more reliable than the direct download? </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/astropy/astropy-tutorials/pull/504/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='5'/>

Reply:
no, it will not.  The archive is the bottleneck, not the access path to the archive.